micronaut-camunda-bpm-feature/build.gradle

```diff
@@ -131,3 +133,22 @@ signing {
 configurations {
     testArtifacts
 }
+
+spotbugs {
+    ignoreFailures = true
```

Comment:
I changed the code
it is configurable
when executed locally the build will fail when there spotbug issues
when executed in GHA the property is set and it will be ignored